docs/OSS Contribution Guide/OSS Contribution Guide - Project Architecture.md

```
@@ -0,0 +1,450 @@
+# OSS Contribution Guide - Project Architecture
+
+Created by: Roberto "Andler" Lucas
+Created time: January 7, 2025 12:17 AM
+Tags: Engineering, Guides, Product
+
+# **Technology Stack - Powering KindFi's Vision**
```

Markdownlint (0.37.0)

Multiple top-level headings in the same document

(MD025, single-title, single-h1)